spec/api/views/players/list_spec.rb

```diff
@@ -0,0 +1,10 @@
+RSpec.describe Api::Views::Players::List, type: :view do
+  let(:exposures) { Hash[format: :html] }
+  let(:template)  { Hanami::View::Template.new('apps/api/templates/players/list.html.erb') }
```

Comment:
Do we want our view to return HTML?

Reply:
@jgayfer ah this is the default test generated by the generate command, I'm going to get into testing today, I'll start with this.

Reply:
Let's fix it before merge 👍